requirements.txt

```diff
@@ -20,4 +20,5 @@ tensorflow-text>=2.13.0
 tensorflow>=2.13.0
 tensorflow-datasets
 tensorboardx
-tensorboard-plugin-profile
\ No newline at end of file
+tensorboard-plugin-profile
+transformer-engine
```

Comment:
hmmm not loving this for all users. Is the solutions a requirements-gpu separately that setup installs for you.

(Weirdly there is no requirements-tpu becuase we get that via jax[tpu])

Reply:
So, the cudnn flash attention uses the transformer-engine, hence it needs to be added as a requirement. Do you recommend having a seprate requirements-gpu?

Reply:
Yes I think we need a requirements-gpu.txt that setup.sh can install additionally.